app.logorrr.LogoRRR.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=LogoRRR
+Type=Application
+Comment=A simple log viewer
+Exec=/app/bin/LogoRRR
```

Comment:
exec is never an absolute path.

Reply:
I changed it to Exec=LogoRRR (although i'm not sure if this works - the binary is linked in /app/bin/LogoRRR in the virtual file system) (sorry if i'm not using the idiomatic wording, I'm new to flathub)